memote-report-app/src/app/report-data.service.ts

```diff
@@ -14,6 +14,7 @@ export class ReportDataService {
   scoredTests: string[] = [];
   reportType: string;
   allExpandState = false;
+  score: any;
```

Comment:
Isn't score to be either `number` or `null`?

Reply:
No, `score` in this case is actually an object. It has the keys "sections" and "total_score" in a snapshot report, and "total_score" in the history report.